diff --git a/node_modules/react-swipeable-views/lib/SwipeableViews.js b/node_modules/react-swipeable-views/lib/SwipeableViews.js
index cdcfaf9..d29ade7 100644
--- a/node_modules/react-swipeable-views/lib/SwipeableViews.js
+++ b/node_modules/react-swipeable-views/lib/SwipeableViews.js
@@ -626,8 +626,8 @@ function (_React$Component) {
     } // eslint-disable-next-line camelcase,react/sort-comp
 
   }, {
-    key: "UNSAFE_componentWillReceiveProps",
-    value: function UNSAFE_componentWillReceiveProps(nextProps) {
+    key: "componentDidUpdate",
+    value: function componentDidUpdate(nextProps) {
       var index = nextProps.index;
 
       if (typeof index === 'number' && index !== this.props.index) {
